src/cow-react/modules/limitOrders/utils/getOrderSurplus.ts

```diff
@@ -2,7 +2,6 @@
 import BigNumber from 'bignumber.js'
```

Comment:
Let's avoid using `bignumber.js` and use `@ethersproject/bignumber`

Reply:
Agree.
But let's refactor when we move this to the SDK?

Reply:
deal